### ISSUE TITLE
This PP is updated ?

### ISSUE THREAD

**Issue by GeoffreyOlivier** [flyfisher604/mpcnc_post_processor#15]
Hello.
I will update my cnc, with an SKR Pro1.2. I wanted to know if this post processor is still relevant for my MPCNC.
Thanks

**Comment by flyfisher604** [comment]
Yes. If the firmware you are running is still based on Marlin, then it is. If you are using V1 Engineering firmware then it is based on Marlin.
